tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -310,6 +312,26 @@ private Grid GetErrorCard(string error, string subError = null)
         return grid;
     }
 
+    private string MergeJsonData(string jsonStringA, string jsonStringB)
+    {
+        if (string.IsNullOrEmpty(jsonStringA))
+        {
+            return jsonStringB;
+        }
+
+        if (string.IsNullOrEmpty(jsonStringB))
+        {
+            return jsonStringB;
+        }
+
+        var objA = JObject.Parse(jsonStringA);
+        var objB = JObject.Parse(jsonStringB);

```

Comment:
Can we use System.Text.Json instead of Newtonsoft here? https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-8-0

Reply:
Ugh, maybe not https://github.com/dotnet/runtime/issues/31433